README.md

```diff
@@ -1,5 +1,20 @@
 # puck
 
+<p align="left">
+  <a aria-label="Measured logo" href="https://measured.co">
+    <img src="https://img.shields.io/badge/MADE%20BY%20Measured-000000.svg?style=for-the-badge&labelColor=000">
+  </a>
+  <a aria-label="NPM version" href="https://www.npmjs.com/package/@measured/puck">
+    <img alt="" src="https://img.shields.io/npm/v/@measured/puck.svg?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="License" href="https://github.com/measuredco/blob/main/LICENSE">
+    <img alt="" src="https://img.shields.io/npm/l/@measured/puck.svg?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="Join the community on Discord" href="https://discord.gg/D9e4E3MQVZ">
+    <img alt="" src="https://img.shields.io/badge/Join%20the%20Discord-blueviolet.svg?style=for-the-badge&logo=Discord&labelColor=000000&logoWidth=20">
```

Comment:
Alternatively we can use the Discord chat widget

<img width="703" alt="image" src="https://github.com/measuredco/puck/assets/985961/4cb1b513-5b1d-4c4d-9161-bcb8b1580243">

Reply:
Think it makes sense to stick with what's there and maybe change to the live widget down the line, if take-up of Discord is good.